app/public/javascripts/timer.js

```diff
@@ -0,0 +1,95 @@
+class Timer {
+    constructor(root) {
+      root.innerHTML = Timer.getHTML();
+  
+      this.el = {
+        minutes: root.querySelector(".timer__part--minutes"),
+        seconds: root.querySelector(".timer__part--seconds"),
+        control: root.querySelector(".timer__btn--control"),
+        reset: root.querySelector(".timer__btn--reset")
+      };
+  
+      this.interval = null;
+      this.remainingSeconds = 120;
+  
+      this.el.control.addEventListener("click", () => {
+        if (this.interval === null) {
+          this.start();
+        } else {
+          this.stop();
+        }
+      });
+  
+      this.el.reset.addEventListener("click", () => {
+        const inputMinutes = prompt("Enter number of minutes:");
```

Comment:
Codacy has a fix for the issue: [Strings must use singlequote.](https://app.codacy.com/gh/cse110-fa21-group17/cse110-fa21-group17/pullRequest?prid=8445086)
```suggestion
        const inputMinutes = prompt('Enter number of minutes:');
```